### ISSUE TITLE
Android Auto doesn't show the fermata auto app

### ISSUE THREAD

**Issue by Alejandro2454** [AndreyPavlenko/Fermata#16]
Dear Andrey:

I've used Fermata Auto perfectly since you made the initial launch. However when upgrading Android auto to version v6.1.610533 and Fermata Auto to version v1.7.3, Android auto on both mobile and car does not show me the Fermata Auto app.  The strange thing is that on Android auto in "version information and permissions" if Fermata Auto appears. I attach screenshot with the above.

Many greetings and attentive to your comments

Alexander

![Screenshot_20210221-174300_Android Auto](https://user-images.githubusercontent.com/64496508/108735950-deed8780-750f-11eb-8f7d-c6a0eacd8919.jpg)

**Comment by AndreyPavlenko** [comment]
Try this tweaker - https://forum.xda-developers.com/t/...er-the-ultimate-android-auto-utility.4194239/

**Comment by Alejandro2454** [comment]
Hi Andrey,
 I have installed and patched fermata auto and the problem remains the same.
 Watch for your comments
 Alexander
![Screenshot_20210222-191820_AA AIO Tweaker](https://user-images.githubusercontent.com/64496508/108778363-078f7480-7544-11eb-8c84-f15ed0707beb.jpg)

**Comment by Alejandro2454** [comment]
Hi Andrey,
 I have installed and patched fermata auto and the problem remains the same.
 Watch for your comments
 Alexander

**Comment by AndreyPavlenko** [comment]
Are you sure v6.1.610533 is not a beta version? The latest AA version I see on the google play is 6.0.6.
The patcher does not work for the beta versions.

**Comment by Alejandro2454** [comment]
Hi Andrey,
 It is the version that is in Google Play from my cell phone.
 I attach images
 Greetings
 Alexander

**Comment by Alejandro2454** [comment]
Hi Andrey,

It may be that this version of AA is here in Chile.
 This version of AA is also in apkmirror in case you can try.

Watch for your comments
 Alexander

https://www.apkmirror.com/apk/google-inc/android-auto/android-auto-6-1-6105-release/#downloads

**Comment by AndreyPavlenko** [comment]
Which version of AIO Tweaker do you use?

**Comment by Alejandro2454** [comment]
I use version v 3.0.1

**Comment by Alejandro2454** [comment]
Hi Andrey,
 In the play store here in Chile it just appeared to me to update the
version v6.1.610543
 What do you recommend me?
 Uninstall auto fermata, update AA, install fermata, apply phenotype
 Watch for your comments.
 Alexander

**Comment by AndreyPavlenko** [comment]
Do not use Phenotype, it doesn't work for AA 6. Use AIO Tweaker instead.

**Comment by Alejandro2454** [comment]
ok Andrey, I'll do it as you say

**Comment by Alejandro2454** [comment]
Hi Andrey,
 AIO Tweaker like phenotype does not work with AA v6.1.610543
 Watch for your comments.
 Alexander

**Comment by Alejandro2454** [comment]
Hi Andrey,

 I'm sorry to bother you.  Do you have any news on how to enable auto
fermata in AA V6, especially the last one which is version v6.1.610543?

 Thanks in advance

 Alexander

**Comment by AndreyPavlenko** [comment]
I've not checked yet, but other users said, that AIO TWEAKER works fine with AA 6.1.6. Please try the latest version https://github.com/shmykelsa/AA-Tweaker/tags

**Comment by AndreyPavlenko** [comment]
I've checked and confirm that AIO TWEAKER works fine with AA 6.1.6.

**Comment by Alejandro2454** [comment]
Hi Andrey,
 Unfortunately for me AIO TWEAKER did not work.
 To fix the problem I tried uninstalling AA and installing an older version
v 6.0 apk.
 However, it did not work since when configuring AA it forced me to update
to the latest version.
 What I did and it worked well for me was to install the apk of version
V5.9.6.  With this AA AIO TWEAKER version it worked very well and I already
have Fermata auto again.
 Very happy
 Greetings
 Alejandro
